<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xVCVSImportPreviewDlg</class>
 <widget class="QDialog" name="xVCVSImportPreviewDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CVS file input preview</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="pInputFileGB">
      <property name="title">
       <string>input</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>input file [txt mode]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="pFileContentBrowser">
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>conversion parameter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="xVPropTable" name="pPropTable">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>25</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="pGenTableGB">
      <property name="title">
       <string>generated table [always in columns]</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTableWidget" name="pDataTable"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QToolButton" name="pHelpTB">
       <property name="text">
        <string>&amp;help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="pUpdateTableTB">
       <property name="text">
        <string>&amp;close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pAcceptTB">
       <property name="text">
        <string>&amp;accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xVPropTable</class>
   <extends>QTableWidget</extends>
   <header location="global">xvproptable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pFileContentBrowser</tabstop>
  <tabstop>pDataTable</tabstop>
  <tabstop>pHelpTB</tabstop>
  <tabstop>pUpdateTableTB</tabstop>
  <tabstop>pAcceptTB</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pUpdateTableTB</sender>
   <signal>clicked()</signal>
   <receiver>xVCVSImportPreviewDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>895</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>769</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pAcceptTB</sender>
   <signal>clicked()</signal>
   <receiver>xVCVSImportPreviewDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>953</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>697</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
